index.html

```diff
@@ -69,7 +69,7 @@ <h2>Getting started</h2>
 
             <h2>Other links</h2>
             <ul>
-                <li><a href="https://twitter.com/freebsdhelp?lang=en" target="_blank">@FreeBSDHelp</a> on Twitter</li>
+                <li>@FreeBSDHelp on <a href="https://twitter.com/freebsdhelp" target="_blank">Twitter</a> | <a href="https://nitter.net/freebsdhelp" target="_blank">nitter</a></li>
```

Comment:
Thanks for this Graham. I'm +1 on a Nitter link for anonymous viewing.

Could you:

 * Move `@FreeBSDHelp on` string into the first `<a></a>`
 * Add `title=`'s to each: `View @FreeBSDHelp on <service>`
 * For the `Nitter` link, add it at the end of the Twitter string in parenthesis

Final string should look like:

 * `Twitter: @FreeBSDHelp (Nitter)`